src/main/java/com/exasol/projectkeeper/validators/dependencies/ProjectDependencyReader.java

```diff
@@ -40,8 +41,9 @@ public ProjectDependencyReader(final MavenFileModelReader fileModelReader,
      * @return list of dependencies
      */
     public List<ProjectDependency> readDependencies(final File pomFile) {
-        final var model = parsePomFile(pomFile);
-        return getDependenciesIncludingPlugins(model).map(this::getDependenciesLicense).collect(Collectors.toList());
+        final var project = parsePomFile(pomFile);
```

Comment:
Would be nice to rename it to `createProjectFromPom` or something like this

Reply:
this method reads the dependencies of a project. So I think the name fits. It uses the project to get it, but that's implementation detail